src/library_nodefs.js

```diff
@@ -1,5 +1,5 @@
 mergeInto(LibraryManager.library, {
-  $NODEFS__deps: ['$FS', '$PATH'],
+  $NODEFS__deps: ['$PATH'],
```

Comment:
> It's ok to have circular deps, but only having circular deps with postsets is the problem, correct?

Yeah, regular ones should not be a problem.

> Then perhaps it could be less confusing to remove the postsets from NODEFS etc., and move them to FS (and keep the circular deps).

Hmm, but `NODEFS` and `TTY` postsets are independent from `FS` - it's `FS` that depends on already having others in its own `postset`.

So moving code of `NODEFS__postset` and `TTY__postset` out of their corresponding declarations would make it somewhat more confusing IMO.

(Or maybe I misunderstand what confusion you want to solve that way)